extensions-utils/src/dependencies/call.rs

```diff
@@ -0,0 +1,43 @@
+use anyhow::Context;
+use fn_error_context::context;
+use std::{
+    ffi::OsStr,
+    path::Path,
+    process::{self, Command},
+};
+
+/// Calls a binary that was delivered with an extension tarball.
+///
+/// # Returns
+/// - On success, returns stdout as a string.
```

Comment:
Is stdout also passed through (displayed) as the binary runs?  If so or if not, can we say here what the behavior is?